engine/src/main/java/org/terasology/engine/math/Side.java

```diff
@@ -44,14 +45,26 @@
     /**
      * @return The horizontal sides, for iteration
      */
-    public static ImmutableList<Side> horizontalSides() {
+    public static List<Side> horizontalSides() {
         return Y_TANGENT_SIDE;
     }
 
+    /**
+     * This provides a static EnumSet of all Sides defined in the enumeration. The result contains the same values as
```

Comment:
The method does not return an EnumSet but an ImutableList. Please adjust the docstring.